.travis/travis_build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+if [ "$TRAVIS_OS_NAME" = "osx" ]
+then
+  cabal install
+else
+  cabal install -f 'static'
+fi
+
+tar -C dist/build/dotenv -cvf dotenv-$TRAVIS_OS_NAME-x86_64-static.tar.gz dotenv
```

Comment:
Should `static` be included in the name for OSX releases too?

Reply:
Good catch @juanpaucar